### ISSUE TITLE
System.Numerics.Matrices projects don't build

### ISSUE THREAD

**Issue by KrzysztofCwalina** [dotnet/corefxlab#159]
When I try to build the main and test projects, I get the following error:
.\corefxlab\src\System.Numerics.Matrices\test\System.Numerics.Matrices.Tests.csproj : error MSB4057: The target "BuildAndTest" does not exist in the project.
.\corefxlab\src\System.Numerics.Matrices\src\System.Numerics.Matrices.csproj : error MSB4057: The target "BuildAndTest" does not exist in the project.

**Comment by KrzysztofCwalina** [comment]
@whoisj, could you take a look at this?

**Comment by whoisj** [comment]
@KrzysztofCwalina I'll look today asap.

**Comment by KrzysztofCwalina** [comment]
@whoisj, thanks for fixing this issue. Unfortunately there is another one: I just updated engineering system to be in sync with CoreFx. The new engineering system does not allow C# 6.0 features. Yet the matrices use string interpolation. I was trying to fix the t4 template during the engineering system update commit, but I am not very familiar with T4 and so I decided to do it as a separate commit. 
If you have time to fix this issue, I would appreciate. If not, I will try to find time to fix it tomorrow.
